src/isort/sorting.rs

```diff
@@ -18,9 +18,12 @@ pub fn module_key<'a>(
 pub fn member_key<'a>(
     name: &'a str,
     asname: Option<&'a String>,
+    order_by_type: bool,
 ) -> (Prefix, String, Option<&'a String>) {
```

Comment:
Hmm, maybe return `(Option<Prefix>, ...)`, and then `if !order_by_type { None } else if name.len() > 1 && string::is_upper(name) { Some(Prefix::Constants) } else if ...`?

Reply:
Oops, it looks like there my branch was out-of-date, sorry about that!
Just rebased- `member_key` is gone and the responsibilities are split between `cmp_members` and prefix`.

Reply:
All good, I should've noticed too, I wrote that newer code 😂